auditbeat/auditbeat.reference.yml

```diff
@@ -845,7 +844,10 @@ setup.kibana:
 #logging.selectors: [ ]
 
 # Send all logging output to syslog. The default is false.
-#logging.to_syslog: true
+#logging.to_syslog: false
+
+# Send all logging output to Windows Event Logs. The default is false.
+#logging.to_eventlog: false
```

Comment:
Not a big fan of naming we have for historical reason and I'm thinking we should change it for 7.0. Perhaps we could already follow here a new scheme similar to what we have in other places:

`logging.eventlog.enabled`. Like this we can have specific logging options under the namespace.

Reply:
I am 100% on board with changing how logging is configured for 7.0. I did it this way to be consistent. I will open an issue to track things that I'd like to "break" for 7.0 (I have a list 😄 ).

Reply:
It should be possible to already do the "new" scheme here as so far it's just a name. More thinking then we have to break one option less for 7 :-) But not sure if we are already settle on if that should be new scheme.

+1 on the issue.

Reply:
Well the scheme I had in mind kind of conflicts. I was thinking of something like.

```
logging.output:
- type: file
   level: info
   json: true
- type: eventlog
  level: warn
```

Reply:
got it, then lets keep it as is in your PR and break it later so we don't need to come to a conclusion now.

I like your format as it gives max flexibility. I assume you have in mind that more then 1 log output can be enabled?

Reply:
Yeah, more than one output and possibly more than one instance of an output type (not 100% sure yet; requires more thought).